composer.json

```diff
@@ -22,8 +22,8 @@
         "sami/sami": "3.3.*"
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.1|^8.0",
         "guzzlehttp/guzzle": "^6.2.3|^7.0.0",
-        "jms/serializer": "^1.6.2"
+        "jms/serializer": "^3.12.0"
```

Comment:
Would it make sense to support both major versions here "^1.6.2|^3.12.0" 

I checked jms/serializer official page and git repository, I see that the support was until 2020.
However there are breaking changes between major versions and maybe some members of our eversign community can not update to 3.x easily.

Reply:
Sure! Updated. :+1: